web3/eth.py

```diff
@@ -200,9 +203,9 @@ def getBlock(
         """
         method = select_method_for_block_identifier(
             block_identifier,
-            if_predefined='eth_getBlockByNumber',
-            if_hash='eth_getBlockByHash',
-            if_number='eth_getBlockByNumber',
+            if_predefined=RPC.eth_getBlockByNumber,
+            if_hash=RPC.eth_getBlockByHash,
+            if_number=RPC.eth_getBlockByNumber,
```

Comment:
@pipermerriam regarding https://github.com/ethereum/web3.py/pull/1520#discussion_r349687149 - thoughts on this general API for wrapping json-rpc endpoints?

Reply:
:+1: for now.  Easy for us to adjust if it proves problematic.